Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+MAINTAINER Canonical
+
+RUN apt-get update -y
+RUN apt-get install -y python3-pip python3-setuptools mysql-server mysql-client libsystemd-dev
```

Comment:
Why mysql-server and mysql-client are required?

Reply:
This is all the packages we are installing today to our application. I can remove them if you think we will not use them. ATM we are not using them.

Reply:
Then remove whatever is not used.